src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -208,42 +222,71 @@ const CustomCollection = ({
   const handleCancelCollection = () => {
     deleteCollectionModal.current?.toggleModal(undefined, false)
   }
+
   // Items to populate dropdown menu
-  const editCustomCollectionItem = (
+  const deleteCustomCollection = (
+    <Button
+      type="button"
+      className={styles.collectionSettingsDropdown}
+      onClick={handleShowRemove}>
+      Delete this collection
+    </Button>
+  )
+
+  const editCustomCollectionTitle = (
+    <Button
+      type="button"
+      className={styles.collectionSettingsDropdown}
+      onClick={() => {
+        setEditing(true)
+        setIsDropdownOpen(false)
+      }}>
+      Edit collection title
+    </Button>
+  )
+  const customCollectionDropdownItems = (
     <>
-      <Button
-        type="button"
-        className={styles.collectionSettingsDropdown}
-        onClick={handleShowRemove}>
-        Delete Collection
-      </Button>
+      <ol>
+        <li>{editCustomCollectionTitle}</li>
+        <li>{deleteCustomCollection}</li>
+      </ol>
     </>
   )
 
+  const handleCancelEdit = () => {
+    if (bookmarks.length === 0) {
+      handleRemoveCollection()
```

Comment:
this logic is confusing to me. Why do we want to remove the collection if there are no bookmarks and they cancel out of editing the title? I thought we are only removing the collection if they don't enter a value for the title (and there wasn't one previously set)